fri/src/prover/mod.rs

```diff
@@ -181,9 +186,18 @@ where
             }
         }
 
+        // compute commitment to last layer
+        match self.folding_factor() {
+            2 => self.build_remainder(channel, &mut evaluations),
+            4 => self.build_remainder(channel, &mut evaluations),
+            8 => self.build_remainder(channel, &mut evaluations),
+            16 => self.build_remainder(channel, &mut evaluations),
+            _ => unimplemented!("folding factor {} is not supported", self.folding_factor()),
+        }
```

Comment:
This seems to be doing the same thing regardless of the branch. Should we change it simply to:
```Rust
self.build_remainder(channel, &mut evaluations);
```

Reply:
Yes, fixed!